src/map/utils/charutils.cpp

```diff
@@ -6150,13 +6150,12 @@ namespace charutils
         // Days start at 0 and the 0th day is Firesday
         // Affinity starts at 0 and the 0th affinity is FIRE_AFFINITY
         // petElement and petElementIdx exist to bridge these gaps here!
-
         auto*   PPet             = static_cast<CPetEntity*>(PChar->PPet);
         ELEMENT petElement       = static_cast<ELEMENT>(PPet->m_Element);
         uint8   petElementIdx    = static_cast<uint8>(petElement) - 1;
         ELEMENT dayElement       = battleutils::GetDayElement();
         WEATHER weather          = battleutils::GetWeather(PChar, false);
-        int16   perpReduction    = PChar->getMod(Mod::PERPETUATION_REDUCTION);
+        int16   perpReduction    = PChar->getMod(Mod::PERPETUATION_REDUCTION) + PChar->PMeritPoints->GetMeritValue(MERIT_SUMMONING_MAGIC_CAST_TIME, PChar);
```

Comment:
Will this change impact all avatars instead of just spirits? It should only impact spirits. Also I think should rename the merit value to describe what it actually does